org.qxmledit.QXmlEdit.yml

```diff
@@ -0,0 +1,46 @@
+app-id: org.qxmledit.QXmlEdit
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: qxmledit
+rename-desktop-file: QXmlEdit.desktop
+rename-appdata-file: QXmlEdit.appdata.xml
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=home
+cleanup:
+  - /include
+
+modules:
+  - name: QXmlEdit
+    buildsystem: qmake
+    config-opts:
+      - QXMLEDIT_DISABLE_SCXML=TRUE
+      - QXMLEDIT_NO_QWTPLOT=TRUE
+      - QXMLEDIT_INST_USE_C11=TRUE
+      - QXMLEDIT_INST_DATA_DIR=${FLATPAK_DEST}/share/qxmledit/
+      - QXMLEDIT_INST_DESKTOP_DIR=${FLATPAK_DEST}/share/applications/
+      - QXMLEDIT_INST_APPDATA_DIR=${FLATPAK_DEST}/share/appdata/
+      - QXMLEDIT_INST_DIR=${FLATPAK_DEST}/bin/
+      - QXMLEDIT_INST_DOC_DIR=${FLATPAK_DEST}/share/doc/qxmledit
+      - QXMLEDIT_INST_LIB_DIR=${FLATPAK_DEST}/lib/
+      - QXMLEDIT_INST_TRANSLATIONS_DIR=${FLATPAK_DEST}/share/qxmledit/translations/
+    post-install:
+      - install -Dm0644 install_scripts/environment/icon/qxmledit.png ${FLATPAK_DEST}/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png
+      - install -Dm0644 install_scripts/environment/icon/qxmledit_48x48.png ${FLATPAK_DEST}/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png
+      - install -Dm0644 src/images/icon.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - desktop-file-edit --set-key="Icon" --set-value="${FLATPAK_ID}" "${FLATPAK_DEST}/share/applications/QXmlEdit.desktop"
+    sources:
+      - type: git
+        url: https://github.com/lbellonda/qxmledit.git
+        tag: 0.9.16
+        commit: 5a06fbcc648500fc0e4c813ffe5d774fa332e6bc
+        x-checker-data:
+          type: git
+          tag-pattern: ^([\d.]+)$
+      - type: patch
+        path: install.patch
+      - type: patch
+        path: categories.patch
```

Comment:
this patch was removed from the repo, so it is not found.

(hence the failure)